android/engine/src/main/java/org/smartregister/fhircore/engine/ui/usersetting/UserSettingScreen.kt

```diff
@@ -62,12 +64,15 @@ import org.smartregister.fhircore.engine.ui.theme.BlueTextColor
 import org.smartregister.fhircore.engine.ui.theme.DividerColor
 import org.smartregister.fhircore.engine.ui.theme.LighterBlue
 
+const val USER_SETTING_ROW_TEST_TAG = "userSettingRowTestTag"
+
 @Composable
 fun UserSettingScreen(
   modifier: Modifier = Modifier,
-  userSettingViewModel: UserSettingViewModel = hiltViewModel()
+  userSettingViewModel: UserSettingViewModel = hiltViewModel(),
```

Comment:
How about we provide the `onEvent` method as a lambda parameter to this function including the rest of the items (properties/functions) need from the `UserSettingViewModel` to be able to test this better? We will not need to provide the `UserSettingViewModel` instance anymore and we can instantiate it on the fragment instead.